.travis.yml

```diff
@@ -4,8 +4,7 @@ os:
   - linux
   - osx
 julia:
-  - 0.6
-  - nightly
+  - 1.0
```

Comment:
Thanks!

Other changes needed:
- [ ] remove `git:   depth: 99999999`
- [ ] add `using Pkg` to after_success scripts
- [ ] delete `cd(Pkg.dir("Pidfile"));`